handlers/mesh.go

```diff
@@ -63,3 +67,13 @@ func MeshGraph(
 		respond(w, code, payload)
 	}
 }
+
+func filterAccessibleControlPlanes(ctx context.Context, namespaceService business.NamespaceService, mesh *models.Mesh) {
+	for i, cp := range mesh.ControlPlanes {
+		// Check if the user is able to access to the control plane
+		_, err := namespaceService.GetClusterNamespace(ctx, cp.IstiodNamespace, cp.Cluster.Name)
+		if err != nil && strings.Contains(err.Error(), "forbidden") {
```

Comment:
iirc you can just check kuberrors.IsForbidden(err) here: https://pkg.go.dev/k8s.io/apimachinery@v0.30.3/pkg/api/errors#IsForbidden.